format-code.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
What's the difference between this file, and [`checkcodeformat.sh`](https://github.com/Seagate/cortx-s3server/blob/4c4a3bf597f599fff53aeb72acbcbe88445864a9/checkcodeformat.sh)? Why would I use this script, and not the existing one?

Reply:
File checkcodeformat.sh just check the code formatting and does not allow editing the code.

Reply:
I think it'd be better to consolidate them into one script then.